pom.xml

```diff
@@ -25,7 +25,8 @@
         <dependency>
             <groupId>org.mongodb</groupId>
             <artifactId>mongo-java-driver</artifactId>
-            <version>2.13.0</version>
+            <version>3.0.0</version>
+            <scope>provided</scope>
```

Comment:
I'd probably remove the provided scope. A project can still exclude the driver and pull in 2.x if they want, but I'm not sure we're super interested in maintaining it such that it works on both